narayana-bom/pom.xml

```diff
@@ -57,11 +57,11 @@
     <version.org.jboss.invocation>2.0.0.Final</version.org.jboss.invocation>
     <version.org.jboss.jandex>2.1.1.Final</version.org.jboss.jandex>
     <version.org.jboss.jboss-transaction-spi>8.0.0.Final</version.org.jboss.jboss-transaction-spi>
-    <version.org.jboss.logging.jboss-logging>3.5.0.Final</version.org.jboss.logging.jboss-logging>
-    <version.org.jboss.logging.jboss-logging-processor>2.2.1.Final</version.org.jboss.logging.jboss-logging-processor>
-    <version.org.jboss.logmanager>1.5.1.Final</version.org.jboss.logmanager>
-    <version.org.jboss.openjdk-orb>9.0.1.Final</version.org.jboss.openjdk-orb>
-    <version.org.jboss.ws>2.0.0.Final</version.org.jboss.ws>
+    <version.org.jboss.logging.jboss-logging>3.6.1.Final</version.org.jboss.logging.jboss-logging>
+    <version.org.jboss.logging.jboss-logging-processor>3.0.2.Final</version.org.jboss.logging.jboss-logging-processor>
+    <version.org.jboss.logmanager>3.0.6.Final</version.org.jboss.logmanager>
+    <version.org.jboss.openjdk-orb>10.1.0.Final</version.org.jboss.openjdk-orb>
+    <version.org.jboss.ws>3.0.0.Final</version.org.jboss.ws>
```

Comment:
I might be wrong but I don't _think_ we are using this property anywhere in Narayana directly?

Reply:
We are using the property to define the jbossws-api version, which is used in multiple places: https://github.com/jbosstm/narayana/blob/main/XTS/localjunit/pom.xml#L54-L58
I'll check what version WildFly is using

Reply:
https://github.com/wildfly/wildfly/blob/main/pom.xml#L561 3.0.0.Final used in WildFly

Reply:
Sorry, I didn't search well enough and the reference to version.org.jboss.ws is in the narayana-bom itself!